modules/dmrpp_module/tests_build_dmrpp/build_dmrpp_macros.m4

```diff
@@ -250,38 +258,39 @@ AS_IF([test -z "$at_verbose"], [
     echo "#       input_file: ${input_file}"
     echo "#         baseline: ${baseline}"
     echo "#           params: ${params}"
+    echo "#      output_file: ${output_file}"
     echo "#         TEST_CMD: ${TEST_CMD}"
 ])
 
 AS_IF([test -n "$baselines" -a x$baselines = xyes],
 [
     AS_IF([test -z "$at_verbose"], [echo "# get_dmrpp_baselines: Calling get_dmrpp application."])
     AT_CHECK([${TEST_CMD}], [], [stdout], [stderr])
-    NORMAILZE_EXEC_NAME([stdout])
-    REMOVE_PATH_COMPONENTS([stdout])
-    REMOVE_VERSIONS([stdout])
-    REMOVE_BUILD_DMRPP_INVOCATION_ATTR([stdout])
+    NORMAILZE_EXEC_NAME([${output_file}])
```

Comment:
NORMALIZE_EXEC_NAME is misspelt (and in the original, too). I'm sure that's my handiwork, but it should be fixed.